src/main/java/pl/edu/agh/toik/ec/migration/MigrationMessage.java

```diff
@@ -4,50 +4,26 @@
 import java.util.List;
 import pl.edu.agh.toik.ec.algorithm.Individual;
 import pl.edu.agh.toik.ec.communication.Message;
+import pl.edu.agh.toik.ec.workers.SimpleMessage;
 
-public class MigrationMessage extends Message {
-    String sender;
-    String receiver;
-    Date timeStamp;
-    List<Individual> content;
+public class MigrationMessage extends SimpleMessage {
+    private List<Individual> individuals;
 
     public MigrationMessage(String sender, String receiver, List<Individual> content, Date timeStamp) {
-        this.sender = sender;
-        this.receiver = receiver;
-        this.content = content;
-        this.timeStamp = timeStamp;
-    }
-
-    public String getSender() {
-        return sender;
-    }
-
-    public void setSender(String sender) {
-        this.sender = sender;
-    }
-
-    public String getReceiver() {
-        return receiver;
-    }
-
-    public void setReceiver(String receiver) {
-        this.receiver = receiver;
-    }
-
-    public Date getTimeStamp() {
-        return timeStamp;
-    }
+        super.setSender(sender);
```

Comment:
is there a reason why `SimpleMessage` has an empty constructor and only setters?